vip_spec.xsd

```diff
@@ -579,6 +580,30 @@ reference elements for polling location
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:simpleType name="DistrictType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="city" />
+      <xs:enumeration value="congressional" />
+      <xs:enumeration value="council" />
```

Comment:
Actually, I might have been a bit too paranoid here.

Our actual logic looks like this:
* Group contests in each dataset by electoral district type (*only* -- other identifiers, like district name, are considered too unreliable across differing data sources; the only thing that's likely to change my mind on that front is if we start getting OCD IDs from many or most VIP participants).

* For each electoral district type, pick the "best" dataset for contests in districts of that type based on request parameters and some heuristics, and serve all the contests from that dataset for that district.

So the cost of having city- and county-council districts share an electoral district type is that we will only serve contests from a single dataset for such a type. In a typical situation (where we have contests from exactly two datasets -- VIP and an unofficial source), this will make little difference in practice: our unofficial dataset always has both city- and county-council contests, and we generally would prefer it for both unless the user has specifically requested only official data, in which case there's no race.

However, as it becomes more common for multiple overlapping VIP datasets to provide information about a single address (for example, some states have a single dataset with statewide contests, and then a dataset for each county), finer-grained district identifiers become potentially more important. (Note that I'd still far prefer relying on OCD IDs than a new custom district identification scheme.) For now, though, it still seems highly unlikely (and suspect) for one VIP dataset  to provide county-council contests and another to provide city-council districts. So I withdraw my objection, unless you can convince the 1622.2 committee to make a last-minute editorial change here.

> Also, how can you ever know in advance that the enum values are sufficiently unique? For example, I'm guessing there is probably more than one kind of "judicial" district in some parts of the country.

You're right, but in practice, we had a quite limited set of district types before and never encountered collisions, at least not for judicial districts. That's a far cry from ultimate safety, but absent emprical evidence of collisions I'm inclined to leave this alone and instead push for better OCD ID adoption.